package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "agentkeepalive": "^4.1.4",
-    "axios": "0.21.1",
+    "axios": "^0.24.0",
```

Comment:
Why not upgrade to 0.21.2?

Reply:
i ran "npm audit fix" that suppose to fix vulnerabilities and thats what it outputed but changed it now manually